changelog.d/20240628_183747_mzhiltso_fix_quality_for_skeletons.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Quality computation for tasks with skeletons and normal labels
```

Comment:
In what way was it broken? Did it crash?

Reply:
Yes, it did crash.

Reply:
Perhaps you could clarify that in the changelog entry?